openpype/hosts/nuke/startup/custom_write_node.py

```diff
@@ -53,24 +58,74 @@
 }
 
 
+class WriteNodeKnobSettingPanel(nukescripts.PythonPanel):
+    """ Write Node's Knobs Settings Panel """
+    def __init__(self):
+        nukescripts.PythonPanel.__init__(self, "Set Knobs Value(Write Node)")
+
+        knobs_value, _ = self.get_node_knobs_setting()
```

Comment:
Shouldn't we name this `knobs_value` to something closer to the true meaning like `preset_name`?

Reply:
is this really resolved? This seems like it is not addressed. Please change the variable name.

Reply:
maybe i just rename it as `Set Preset(Write Node)`